ipa-core/src/app.rs

```diff
@@ -160,12 +167,21 @@ impl HelperApp {
 }
 
 #[async_trait]
-impl RequestHandler for Inner {
-    type Identity = HelperIdentity;
+impl RequestHandler<ShardIndex> for Inner {
+    async fn handle(
+        &self,
+        _req: Addr<ShardIndex>,
+        _data: BodyStream,
+    ) -> Result<HelperResponse, ApiError> {
+        Ok(HelperResponse::ok())
```

Comment:
It is more future proof to panic by default - returning an ACK for all request is a bad response